tests/flow/test_concurrent_query.py

```diff
@@ -236,3 +250,40 @@ def test_05_concurrent_read_delete(self):
         # Make sure Graph is empty, e.g. graph was deleted.
         resultset = graphs[0].query("MATCH (n) RETURN count(n)").result_set
         self.env.assertEquals(resultset[0][0], 0)
+
+    def test_06_concurrent_write_delete(self):
+        redis_con = self.env.getConnection()
+        heavy_write_query = """UNWIND(range(0,999999)) as x CREATE(n)"""
+        threads = []
+        t = threading.Thread(target=thread_try_run_query, args=(graphs[0], heavy_write_query, 0))
+        t.setDaemon(True)
+        threads.append(t)
+        t = threading.Thread(target=thread_delete_graph, args=(redis_con, GRAPH_ID))
+        t.setDaemon(True)
+        threads.append(t)
+        for thread in threads:
+            thread.start()
+        for thread in threads:
+            thread.join()
```

Comment:
There are only 2 threads, maybe we should name them, writer and delete thread.

Reply:
done